src/PaymentMethods/BankTransfer.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * The Zaver Checkout payment gateway.
+ *
+ * @package ZCO/PaymentMethods
+ */
+
+namespace Zaver\PaymentMethods;
```

Comment:
Change the namespace to include the `Krokedil` part before `Zaver`, and do this for all other gateways under the src folder as well.